knowledge-base/adr/0031-resilience-options.md

```diff
@@ -374,28 +232,25 @@ executeHttpRequest(
     destinationName: 'my-dest'
   },
   {
-    timeout: 456,
-    resilience: resilience({ timeout: () => 123 })
+    timeout: 456 //deprecated
   }
 );
-
-//use 123 as timeout
 ```
 
-In the intermediate period where options can be given via the old and new we have to do evaluation:
+From version 3 on there will be the middleware approach to add resilience.
```

Comment:
[q] I guess the deprecation period will not happen due to it being a breaking change, so this should be removed, correct?